1-introduction/introduction_Task2.py

```diff
@@ -1,11 +1,23 @@
+from typing import List
+
 numbers = [1, 2, '0', '300', -2.5, 'Dog', True, 0o1256, None]
-for n, i in enumerate(numbers):
-    try:
-        numbers[n] = int(i)
-    except TypeError:
-        numbers.pop(n)
-    except ValueError:
-        numbers.pop(n)
-
-print(f'Maximum value is: {max(numbers)}')
-print(f'Minimum value is: {min(numbers)}')
+
+
+def get_min_max(li: List) -> None:
```

Comment:
It is good practice not to print the result of a function in the function itself, but to output the result and then print it to the screen (except in rare cases or for debugging).

Reply:
It will be problematic to test it if unit tests are created

Reply:
Same for a few other task.